_includes/templates/summaries/provide-feedback.html

```diff
@@ -0,0 +1,18 @@
+<h2 class="h4">Provide Feedback</h2>
+<p class="short-desc">
+    We take pride in our feedback-centered culture
+    that helps us recruit great people.
+    If you have any feedback, or any specific questions, let us know at.
```

Comment:
I agree. Maybe I misheard @sarahsimpson09 .. can we take @jimmynotjim suggestion?